code/src/components/TodoList.js

```diff
@@ -0,0 +1,198 @@
+import React from "react";
+import { useSelector, useDispatch } from "react-redux";
+import styled from "styled-components";
+import moment from "moment";
+
+import todos from "../reducers/todos";
+import AddTodo from "./AddTodo";
+import Header from "./Header";
+
+const TodoForm = styled.form`
+  position: relative;
+  display: flex;
+  margin: 20px;
+  padding: 10px;
+  border-bottom: 1px solid pink;
+  min-height: 30px;
+  width: 200px;
+  margin: 20px auto;
+  text-align: center;
+  align-items: center;
+
+  @media (min-width: 768px) {
+    min-height: 40px;
+    width: 400px;
+  }
+
+  p {
+    overflow-wrap: break-word;
+    max-width: 130px;
+    font-size: 15px;
+    text-transform: uppercase;
+    @media (min-width: 768px) {
+      max-width: 310px;
+      font-size: 18px;
+    }
+  }
+`;
+const ButtonTrash = styled.button`
+  position: absolute;
+  display: flex;
+  right: 7px;
+  font-size: 22px;
+  font-weight: bold;
+  border: none;
+  color: #cf7b7b;
+  background-color: rgba(235, 227, 227, 1);
+  margin-top: auto;
+
+  &:hover {
+    color: red;
+  }
+`;
+const Label = styled.label`
+display: flex;
+  margin-top: auto;
+  position: absolute;
+  right: 40px;
+  
+  
+  ${(props) =>
+    props.complete === true
+      ? `text-decoration: line-through`
+      : `text-decoration: none`}
+
+}
```

Comment:
I think props is hard to understand.. So I can't really understand this:) if ypu have time some time please explain.